grails-app/controllers/org/pih/warehouse/api/StockMovementItemApiController.groovy

```diff
@@ -40,7 +40,13 @@ class StockMovementItemApiController {
     }
 
     def getStockMovementItems() {
-        List<StockMovementItem> stockMovementItems = stockMovementService.getStockMovementItems(params.id, params.stepNumber, params.max, params.offset)
+        String stockMovementId = params.get("id")
+        String stepNumber = params.get("stepNumber")
+        String max = params.get("max")
+        String offset = params.get("offset")
+        Boolean createNew = params.boolean("createNew", true)
```

Comment:
I would suggest a command object for that instead of having to do it manually (even if you have to pass that many arguments to the method below and would have to use getters for that).